microshift_cli_ref/microshift-cli-using-oc.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-The optional OpenShift CLI (`oc`) tool is available for you to use if you are already familiar with {OCP} and Kubernetes.
+If you are already familiar with {OCP} and Kubernetes, the optional OpenShift CLI (`oc`) tool may be especially useful. The subset of `oc` commands relevant for single-node {product-title} deployments is enabled.
```

Comment:
I'm not sure what this means?

> The subset of `oc` commands relevant for single-node {product-title} deployments is enabled.

Reply:
I'm not sure why we need to say anything about the number of nodes at all. Some of the APIs that oc uses are not present in MicroShift, but that's not because of the node count. Are you just looking for a way to express the disclaimer that some oc commands do not work with MicroShift?

Reply:
If there is a specific way, then yes. If there isn't any way to classify what doesn't work, there's no need.

Reply:
The best we can say is that the commands that use APIs that are present work. Which is not very helpful if you don't know what APIs are used by commands.

We did go through each of the commands and exclude the ones that do not work, so only the relevant ones show up in the oc docs in the MicroShift section. Maybe that's good enough?